### ISSUE TITLE
Deposit transaction ID - Transaction not found

### ISSUE THREAD

**Issue by bitsanity** [bisq-network/bisq#3305]
re: https://bisq.community/t/deposit-transaction-id-transaction-not-found/8424

Quoting huey:

> There’s an issue that has bee causing an error when broadcasting transactions. In this case, it seems that the make offer transaction wasn’t successfully broadcast but the Bisq Network still accepted it as a valid offer.

Details:

Trade is stuck on Step 1 “Wait for blockchain confirmation”. The Deposit and Maker fee transaction IDs shown are not found by block explorer(s).

Version: Bisq-64bit-1.1.6.deb
Trade ID: efeikv2-bfc01e8c-6541-49ed-8c7d-ff1f17c21877-115
Deposit txid: 4e96b45716582fc18b2ab8aabceaaa5c450412329644858625ed4966dcf84ab6
Maker txid: c10e3ecf7a516b687afefcbf86d00c26497f84dcc7640d7a7d5d998a5d40d179
Taker txid: 4df815d6a4b6250d965a40dfa7118e023b74e0f1dc1663401932c05ae2f5665e

Only the Taker txid is recognized by block explorers.

**Comment by huey735** [comment]
@sqrrm @ripcurlx 
This seems like an exceptional error of the trade protocol. The offer shouldn't have made to the offer book without a confirmation of the makeOfferTx.

Related to https://github.com/bisq-network/bisq/issues/3306

**Comment by bitsanity** [comment]
Something still broken - transactions are not making it to the blockchain.

I see the dispute payout transaction from earlier is still not found on blockchain:
https://www.blockchain.com/btc/tx/8391d4799bffdd9d80349e5d4324eac5b45fa77179ac340bcd8cdb36f2f34cfc

Without realizing, I tried to send bitcoin from my Bisq wallet and it failed to appear in the mempool in time.

https://www.blockchain.com/btc/tx/2642b5e520bd407d2385bf6a60f8d7492a0ad76d5333959a73ba215bde390b8a

My log entries include:
fee 0.0000815 BTC for 813 bytes (10 Satoshi/Byte)
which looks reasonable (about $0.81 USD)

I see the onion stuff and peer messaging going on. There is an entry:

Sep-22 14:02:25.383 [JavaFX Application Thread] INFO o.b.c.TransactionBroadcast: broadcastTransaction: We have 9 peers, adding 2642b5e520bd407d2385bf6a60f8d7492a0ad76d5333959a73ba215bde390b8a to the memory pool

but the transaction is still not in mempool.

The log file is already 6MB so have extracted what I think is relevant time period below. Please let me know if I can add anything to the issue. Thanks.

edit: remove log details for privacy and it sounds like wiz may have found something already

**Comment by wiz** [comment]
@bitsanity are you using custom bitcoin nodes in your settings? did you delete your SPV file and re-sync the blockchain?

**Comment by bitsanity** [comment]
@wiz no custom settings at all. I have not deleted SPV file yet - is it safe to just delete it and rerun bisq? And to be sure, is this the SPV file to delete:

$HOME/.local/share/Bisq/btc_mainnet/wallet/bisq.spvchain

Thanks

edit: I did the resync and the pending transactions are now gone, balance looks ok :)

**Comment by huey735** [comment]
@bitsanity for future reference you can [resync the spv chain](https://docs.bisq.network/backup-recovery.html#resync-spv-chain) from inside the software.

**Comment by huey735** [comment]
Matter of fact @bitsanity if you don't mind could you flag some of the transactions of yours that weren't going through? I assume `2642b5e520bd407d2385bf6a60f8d7492a0ad76d5333959a73ba215bde390b8a` is one of them, right?

**Comment by bitsanity** [comment]
@huey735 thanks for the note about the resync button. I did that, cleared old tor files, then it started again and completed successfully.

Yes that (2642b5...) was the attempted withdrawal. There were two previous ones stuck as well. I attempted sale of 0.065 BTC on Sep21. The taker's transaction made it, but my maker fee and deposit got stuck.

deposit: 4e96b45716582fc18b2ab8aabceaaa5c450412329644858625ed4966dcf84ab6

Here is the deposit transaction log part:

edit: deleted transaction log details for privacy and as wiz seems to have found something already
